src/FinishedList.js

```diff
@@ -0,0 +1,49 @@
+
+
+function FinishedList({location, initialState, onClickTodoItem, onDeleteTodoItem}) {
+
+    this.state = {...initialState};
+
+    this.setState = (nextState) => {
+        this.state = {
+            ...this.state,
+            ...nextState
+        }
+
+        this.render();
+    }
+
+    this.render = (finishedTodo) => {
+
+        const p = document.querySelector('.finished-todo-list-count');
+        p.textContent = finishedTodo.length;
+
+        location.innerHTML = '';
+
+        finishedTodo.map(todo => {
+            const li = document.createElement('li');
+            const btn = document.createElement('button');
+
+            li.className = "todo-middle-text todo-item finished-todo-item";
+            li.id = todo.todoID;
+            btn.className = "delete-btn";
+
+            li.textContent = todo.todoContent;
+            li.appendChild(btn);
+
+            li.addEventListener('click', (e) => {
+                if (e.target.className !== 'delete-btn') {
+                    onClickTodoItem({id: todo.todoID, todoContent: todo.todoContent});
+                }   else {
+                    onDeleteTodoItem(todo.todoID);
+                }
```

Comment:
사소한 부분일수도 있으나, 보통 if문의 조건엔 `!==`보다 `===`를 사용하는 것이 것이 가독성에 더 좋다고 합니다!
```suggestion
                if (e.target.className === 'delete-btn') {
                    onDeleteTodoItem(todo.todoID);
                }   else {
                    onClickTodoItem({id: todo.todoID, todoContent: todo.todoContent});
                }
```